docs/toc.ts

```diff
@@ -72,7 +71,6 @@ const toc: RawTableOfContents = {
         link: "latest",
         pages: [
           ["modifying-the-head", "Modifying the <head>", "link:latest"],
-          ["setting-the-language", "Setting the language", "link:latest"],
```

Comment:
Is setting the language now such an obvious thing that the documentation should be deleted? Or could it be beneficial to completely redo the example to account for the new functionality?